sesman/chansrv/clipboard_file.c

```diff
@@ -412,15 +453,17 @@ clipboard_send_file_data(int streamId, int lindex,
     fd = g_file_open_ex(full_fn, 1, 0, 0, 0);
     if (fd == -1)
     {
-        LOG_DEVEL(LOG_LEVEL_ERROR, "clipboard_send_file_data: file open [%s] failed",
-                  full_fn);
+        LOG(LOG_LEVEL_ERROR, "clipboard_send_file_data: file open [%s] failed: %s",
+            full_fn, g_get_strerror());
+        clipboard_send_filecontents_response_fail(streamId);
         return 1;
     }
     if (g_file_seek(fd, nPositionLow) < 0)
     {
         LOG_DEVEL(LOG_LEVEL_ERROR, "clipboard_send_file_data: seek error "
```

Comment:
The logging call for a seek fail should definitely be a LOG() rather than a LOG_DEVEL(). Suggest also using g_get_strerror() to get the error from the seek operation.